exchanges/bitflyer/bitflyer_types.go

```diff
@@ -294,3 +300,23 @@ type NewOrder struct {
 	MinuteToExpire float64 `json:"minute_to_expire"`
 	TimeInForce    string  `json:"time_in_force"`
 }
+
+// Time is a custom type that wraps time.Time to implement json.Unmarshaller
+type Time time.Time
```

Comment:
🥃 🦘 ( Watering hole )
I liked the way we named `types.DateTime`.
This format below isn't insane, but it's also not covered by an RFC or 8601 that's easily nameable (that I could find).
It seems like iso-8601 without timezone but with `.` nanoseconds `ISO8610NoTZDotNS`
Wondering if we just move it to types.Time.